charts/latest/spdk-csi/templates/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -0,0 +1,139 @@
+{{- if .Values.snapshotclass.create -}}
```

Comment:
can we have all the crds in a single file?

Reply:
We can separated with `---`

Reply:
But it will make the single file ambiguous